Templates/CSharp/EdgeDriverTemplate/ProjectTemplates/CSharp/Test/EdgeDriverTest.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Edge;
+
+namespace $safeprojectname$
+{
+    [TestClass]
+    public class EdgeDriverTest
+    {
+    // In order to use web driver of Edge, it is required to download and install the Edge Web Driver from
+    // https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
+    // Then add it to your PATH enviroment variable and run it at the background.
+        EdgeDriver Driver;
```

Comment:
Fix the spacing for the comments.
And "Driver" variable should be lower cased as it is variable and not a property.
Also add a new line have the declaration

Reply:
Sure, will fix it in next commit